server/config/environment/index.js

```diff
@@ -44,10 +44,11 @@ var all = {
     elasticTypes: elasticTypes.TYPE_MAP_ARRAY,
     annotationIndex: process.env.ANNOTATION_INDEX || 'dig-annotations',
     annotationType: process.env.ANNOTATION_TYPE || 'annotation',
+    memexUser: process.env.MEMEX_USER|| '',
```

Comment:
Could you please change MEMEX_USER and MEMEX_PASSWORD to a single authentication variable?  We'll always use them together and we won't use one without the other.  Additionally, can you use a more specific name, since the name it MEMEX is very generic and could apply to anything on the project?  Maybe `IMAGE_SERVICE_AUTH`